fhir_py_types/ast.py

```diff
@@ -28,11 +28,14 @@ class AnnotationForm(Enum):
 def make_type_annotation(
     type_: StructurePropertyType, form: AnnotationForm
 ) -> ast.expr:
+    name = type_.code
+    if name == "Resource":
+        name = "AnyResource"
```

Comment:
Workaround to replace "Resource" with the union of all resources which is "AnyResource"